apps/rush-lib/src/api/EnvironmentConfiguration.ts

```diff
@@ -89,7 +89,12 @@ export const enum EnvironmentVariableNames {
    *
    * POSIX is a registered trademark of the Institute of Electrical and Electronic Engineers, Inc.
    */
-  RUSH_GLOBAL_FOLDER = 'RUSH_GLOBAL_FOLDER'
+  RUSH_GLOBAL_FOLDER = 'RUSH_GLOBAL_FOLDER',
+
+  /**
+   * Provides a credential for a remote build cache, if configured.
+   */
+  RUSH_BUILD_CACHE_CREDENTIAL = 'RUSH_BUILD_CACHE_CREDENTIAL'
```

Comment:
Add `@remarks` to explain:
- What is the syntax for a valid credential string
- Give a realistic example value
- Provide a link to the Azure docs with the details about how these credentials work and how to create them

Reply:
(If other services like AWS might have a substantially different credential, I wonder if we should have a different environment variable for each provider. For example `RUSH_AZURE_CREDENTIAL`.  It is easier to document something specific, versus a generic variable that can have arbitrary things stuffed into it.)